posts/2022-01-31-multi-party-data-sharing-with-control.md

```diff
@@ -0,0 +1,271 @@
+---
+title: "Multi-Party Data Sharing, with Control"
+description: "How Vendia Share's access controls enable trusted and secure real-time data sharing among partners"
+date: '2022-01-31'
+categories:
+- architecture
+- concepts
+- tutorials
+authors:
+- James Gimourginas
+---
+
+## Overview
+For our customers, multi-party data sharing means many participants transacting with each other in real time.  While this sounds simple enough, the reality is getting participants to _agree to collaborate_ can be a challenge.  Many of these participants (often separate organizations) have grown accustomed to under-sharing their data, either because of the cost and complexity or because of insufficient access controls to truly share with trust.  Vendia Share makes trusted real-time data sharing possible thanks to the concept of a Universal Application ("Uni" for short).  A Uni is composed of one or more nodes, and all nodes include the single source of truth for all participants.  It's common for each participant to have its own node, which allows each participant to tailor the access controls to its node and to the data written to it.
```

Comment:
`all nodes include the single source of truth for all participants`

This seems slightly contradictory since there are multiple copies of the single source of truth, and each node could have a different view on that source of truth.

Reply:
This is an interesting observation. Maybe another way to look at it is to think of the source of truth as, "A fact stored in the Uni(network of nodes) is the same for all. But, a fact may not be visible to all". If in our results we show that queries match more results than reported, it will mean more facts exists but we can't show you everything because of permission restrictions. This will keep Uni true to the definition of "source of truth".

Reply:
I'll rework this section to address the contradiction.  Will be more accurate, and a good preview of later sections.